test/runtests.jl

```diff
@@ -3112,7 +3114,71 @@ else  # run HiGHS tests
                 end
                 @test results["ElectricTariff"]["year_one_demand_cost_before_tax"] ≈ demand_charge_expected atol=1E-6
             end
+        end
+
+        @testset "Align load profiles based on load year" begin
+            """
+            Common use case: ElectricLoad.loads_kw is input with specific year, but heating and/or cooling is 
+                simulated with either a schedule-based FlatLoad, or b) CRB type with annual or monthly energy
+            This test confirms that the simulated FlatLoad type and CRB are shifted to start on Monday for 2024
+            """
         
+            input_data = JSON.parsefile("./scenarios/load_year_align.json")
+            year = 2024
+            # ElectricLoad.loads_kw is 2024, and heating and cooling loads are shifted to align
+            # Use a FlatLoad_16_5 shifted to 2024 (Monday start) with the web tool's custom load builder
```

Comment:
this isn't that important, but what does this comment mean? That the flat load has already been shifted after getting the 2017 version from the web tool?

Reply:
I just built a load with the year=2024, and the web tool aligned the weekday/weekend profiles to the 2024 year.